WireAnalytics/Sources/WireAnalytics/AnalyticEvents/AppOpenAnalyticsEvent.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+/// Struct representing an analytics event for app opening.
+public struct AppOpenAnalyticsEvent: AnalyticsEvent {
+
+    public init() {}
+
+    /// The name of the event.
+    public var eventName: String {
+        "appOpen"
```

Comment:
According to the spec this should be `app.open`.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/9b3bf246ac667732625e4d49a0820cf15fd9dd37